spaces/yaxis/index.json

```
@@ -1,5 +1,5 @@
 {
-  "token": "0xb1dC9124c395c1e97773ab855d66E879f053A289",
+  "token": "0x162c07f7076b5287cf1d599972c480c51ae00743",
   "name": "yAxis",
   "network": "1",
   "symbol": "YAX",
```

If you want a custom domain you will need to add the `domain` field in `index.json`. https://docs.snapshot.page/guides/create-a-space#3-space-metadata
```suggestion
  "symbol": "YAX",
  "domain": "gov.yaxis.io",
```

---

We are doing some changes for spaces settings, you can change your strategy token if you want but the root token value must stay like that.